lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex

```diff
@@ -79,10 +79,9 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackSender do
   @impl true
   def handle_prepared_to_playing(ctx, state) do
     actions =
-      Enum.flat_map(ctx.pads, fn
-        {Pad.ref(:output, _ref) = pad, _pad_data} -> [caps: {pad, %Membrane.RTP{}}]
-        _other -> []
-      end)
+      ctx.pads
+      |> Stream.filter(fn {_pad_id, %{name: name}} -> name == :output end)
```

Comment:
There's no point in using `Stream`, it will make it slower. You can Google Enum vs Stream, stream might make sense if going through a large collection and need to avoid intermediate collections or use only a few results e.g. `Enum.take(5)`
Here's a simple benchmark I wrote:
```elixir
Mix.install([{:benchee, "~> 1.0"}])

encoding = "h"

pads = %{
  {:input, 1} => %{name: :input, sos: true, eos: false, caps: %{}},
  {:output, 0} => %{name: :output, sos: true, eos: false, caps: %{}},
  {:output, 1} => %{name: :output, sos: true, eos: false, caps: %{}},
  {:output, 2} => %{name: :output, sos: true, eos: false, caps: %{}},
  {:output, 3} => %{name: :output, sos: true, eos: false, caps: %{}},
  {:output, 4} => %{name: :output, sos: true, eos: false, caps: %{}}
}

Benchee.run(
  %{
    stream: fn ->
      pads
      |> Stream.filter(fn {_pad_id, %{name: name}} -> name == :output end)
      |> Enum.flat_map(fn {pad, _pad_data} ->
        [caps: {pad, MapSet.new()}, event: {pad, %{variant: encoding}}]
      end)
    end,
    enum: fn ->
      pads
      |> Enum.filter(fn {_pad_id, %{name: name}} -> name == :output end)
      |> Enum.flat_map(fn {pad, _pad_data} ->
        [caps: {pad, MapSet.new()}, event: {pad, %{variant: encoding}}]
      end)
    end
  },
  warmup: 5
)
```

```
Operating System: Linux
CPU Information: AMD Ryzen 7 PRO 4750U with Radeon Graphics
Number of Available Cores: 16
Available memory: 29.11 GB
Elixir 1.14.0
Erlang 25.0

Benchmark suite executing with the following configuration:
warmup: 5 s
time: 5 s
memory time: 0 ns
reduction time: 0 ns
parallel: 1
inputs: none specified
Estimated total run time: 20 s

Benchmarking enum ...
Benchmarking stream ...

Name             ips        average  deviation         median         99th %
enum        510.13 K        1.96 μs  ±1198.67%        1.89 μs        3.35 μs
stream      458.63 K        2.18 μs   ±881.20%        1.89 μs        3.84 μs

Comparison:
enum        510.13 K
stream      458.63 K - 1.11x slower +0.22 μs
```